client/src/components/mainContent.jsx

```diff
@@ -151,6 +178,13 @@ const useStyles = makeStyles((theme) => ({
     fontWeight: 500,
     color: "black",
   },
+  DividerInCardContent: {
+    "margin-top" : "30px",
+    "margin-bottom" : "30px"
```

Comment:
```suggestion
    marginBottom : "30px"
```

Reply:
another way to write this if you want to save on num of lines of code

```
margin: `30px auto`
```

this will add 30px top and bottom